web-ui/controller/badges.ml

```diff
@@ -1,5 +1,5 @@
 type schema = {
-  schema_version : int; [@key "schemaVersion"]  (** always: 1 *)
+  schema_version : int; [@key "schemaVersion"] [@default 1]  (** always: 1 *)
```

Comment:
```suggestion
  schema_version : (int  [@default 1]); [@key "schemaVersion"]  (** always: 1 *)
```
In the [ppx_yojson README](https://github.com/ocaml-ppx/ppx_deriving_yojson#default) it is written this way. I don't know if it changes something, but I thought it worth pointing out...

Reply:
Thanks - I should have read that first :)